modules/ROOT/pages/connected-apps-overview.adoc

```diff
@@ -16,6 +16,25 @@ OAuth (Open Authorization) is an open protocol that provides secure API authoriz
 OAuth can authorize access to resources without revealing user credentials to apps.
 OpenID Connect identifies the end user and obtains information to pass to OAuth 2.0 connected apps.
 
+
+// Sara add some indentation or mark up here please
+
+In order to understand the purpose of Connected Apps, there are several technical concepts and terms to be familiar with, beginning with OAuth 2.0 (commonly referred to as just OAuth) and OpenID Connect (commonly referred to as OICD). Before breaking down these terms, it’s important to understand that these flows are completed in order to give access to Anypoint APIs to only those services and users with the proper permissions. 
+
+OAuth 2.0
+When understanding OAuth 2.0, an authorization protocol, we must first define several key terms to ground our understanding. The first is the client, which refers to the resource that wants the data and could be a computer or server. Next, the authorization server refers to the server that can authorize, and the resource server refers to the system with the data. The access token is the key to get into data in the resource server and the redirect URI is where you go if the user or service authorizes the other service.
+
+There are two types of flows for OAuth 2.0. 
+
+The first is the authorization_code flow. In this flow, the client goes to the authorization server and and the server pops up a window asking for authorization for specific scopes from the user. If the user authorizes the client to have this information, an authorization code is sent the redirect URI. The redirect URI then exchanges the authorization code for an access token from the authorization server which is then taken to the resource server to obtain the data by talking to a specific API. This is called an application acting on behalf of the user.
+
+The second flow is the client_credentials flow. In this flow, each piece is very similar but instead of a user authorizing the application, the application can authorize on its own behalf. This is also called a service-to-service use case.
```

Comment:
```suggestion
`client_credentials`::
In the client_credentials flow, the client navigates to the authorization server. The server then requests authorization for specific scopes, or _permissions_, from the application. If the application authorizes the client to have this information, an authorization code is sent the redirect URI. The redirect URI then exchanges the authorization code for an access token from the authorization server. The access token is then sent to the resource server to obtain the data by talking to a specific API. This flow demonstrates a _service-to-service_ use case.
```

Reply:
Better to restate everything than rely on legacy content in case something is deprecated. I doubt that would happen here, but it is good practice.